api/src/main/java/org/openmrs/module/reporting/cohort/definition/EncounterCohortDefinition.java

```diff
@@ -346,4 +349,16 @@ public Date getCreatedOnOrAfter() {
 	public void setCreatedOnOrAfter(Date createdOnOrAfter) {
 		this.createdOnOrAfter = createdOnOrAfter;
 	}
+
```

Comment:
We seems to be repeating these a lot.  I wonder if we could move these into a base abstract class (so they don't need to be repeated) or an interface to ensure consistency.